src/compiler/gcc.rs

```diff
@@ -275,6 +283,11 @@ where
                     OsString::from(arg.flag_str().expect("Compilation flag expected"));
             }
             Some(ProfileGenerate) => profile_generate = true,
+            Some(ProfileUse(_)) => {
+                if compiler_kind != CCompilerKind::Clang {
+                    cannot_cache!(arg.flag_str().unwrap(), "only Clang supported".into())
```

Comment:
Just use a string literal here in `cannot_cache!`, since we use the strings as buckets to count non-cacheable compile requests and output them in the stats. You could use "-fprofile-use but not clang" or something like that.